docs/content/any/project/changelogs/NEXT.md

```diff
@@ -10,6 +10,13 @@ draft: true
 
 ## `RELEASED_PACKAGE_1` NEW_VERSION
 
+### Java
+
+#### Other bugs & improvements.
+
+A new `loadFilesFromResources` API has been added to allow loading policy source code from resource files contained in your packaged `.jar`. Special thanks to [kovacstamasx](https://github.com/kovacstamasx) for this contribution.
```

Comment:
```suggestion
A new `loadFilesFromResources` API has been added to allow loading policy source code from resource files contained in your packaged `.jar`. Special thanks to [`@kovacstamasx`](https://github.com/kovacstamasx) for this contribution.
```

Always best to put the github username as a code literal to skip the spell checker :)

Reply:
TIL of this trick!